.pre-commit-config.yaml

```
@@ -17,12 +17,12 @@ repos:
       - id: tox-ini-fmt
         args: ["-p", "fix"]
   - repo: https://github.com/tox-dev/pyproject-fmt
-    rev: 1.5.1
+    rev: 1.5.3
     hooks:
       - id: pyproject-fmt
         additional_dependencies: ["tox>=4.11.3"]
   - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: v3.1.0
+    rev: v4.0.0-alpha.3
```

Can we revert this alpha change and land the rest first?
```suggestion
    rev: v3.1.0
```